test/hotspot/jtreg/runtime/coroutine/jniMonitorExitTest.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
```

Comment:
why?

Reply:
This test will use jni to get an object named lock in class SelectorProvider, but this object was deleted in https://bugs.openjdk.org/browse/JDK-8236246. So we delete all those tests related.